src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -609,7 +609,7 @@
         "DELETE_ENABLED": "Enabled this policy",
         "NEW": "New",
         "NAME_TOOLTIP": "replication rule name should be at least 2 characters long with lower case characters, numbers and ._- and must be start with characters or numbers.",
-        "DESTINATION_NAME_TOOLTIP": "Destination name should be at least 2 characters long with lower case characters, numbers and ._-/ and must be start with characters or numbers.",
+        "DESTINATION_NAME_TOOLTIP": "Destination name should be at least 2 characters long with lower case characters, numbers and ._- and must be start with characters or numbers.",
```

Comment:
The character `/` isn't allowed for the destination namespace?

Reply:
Yes
![image](https://user-images.githubusercontent.com/30999793/121619125-a069e180-ca9a-11eb-9e88-a53b20c8cb57.png)

Reply:
a/b is allowed. And Modified i18n